TeamCode/src/main/java/com/acmerobotics/relicrecovery/motion/MotionConstraints.java

```diff
@@ -0,0 +1,58 @@
+package com.acmerobotics.relicrecovery.motion;
+
+/**
+ * @author kellyrm
+ * constraints for a motion profile
+ */
+
+public class MotionConstraints {
+
+    /**
+     * max absolute value of velocity
+     */
+    public double maxV;
```

Comment:
Should these be `final`? Just a thought.

Reply:
that sounds good
what values should be assigned in the default constructor?

Reply:
I think it might be better to just omit the default constructor.